cove/lib/common.py

```diff
@@ -799,11 +799,30 @@ def get_additional_codelist_values(schema_obj, json_data):
             path_string = '/'.join(path_no_num)
 
             if path_string not in additional_codelist_values:
+                
+                codelist_url = schema_obj.codelists + codelist
+                codelist_add_url = None
```

Comment:
There is nothing to stop there being multiple add, sub urls. So this should be a list.  I think add/sub should be a single list as order matters.